components/menu/demo/menu.html

```diff
@@ -158,6 +158,9 @@ <h2>Supporting Slot</h2>
 						<d2l-menu-item-link text="Dwarf Planets (link)" href="https://en.wikipedia.org/wiki/Dwarf_planet">
 							<span slot="supporting">39+ AU</span>
 						</d2l-menu-item-link>
+						<d2l-menu-item-link text="Dwarf Planets (link to download)" href="../menu.js" target="_blank" download>
```

Comment:
This seems a little strange - the `download` attribute is being used here as if the property is defined as a `Boolean`, but the component defines the property as a `String`.  This relates to my other common about the type and whether the value is required or not.  Curious what gets passed down in this case.

Reply:
I think that's OK based [on the docs](https://developer.mozilla.org/en-US/docs/Web/HTML/Element/a) of how it's supposed to work.

Reply:
The attribute is a string. Lit will initialize strings as empty-string if no value is passed